src/skins/index.ts

```diff
@@ -0,0 +1,33 @@
+import { IAppSkin } from "./skin.model";
+import { arrayToHashmap } from "src/app/shared/utils";
+
+// TODO: Eventually these skins should be authored from templates
```

Comment:
I think probably the skins should also be kept in the deployment folder, so that the various plh apps define their own set of skins available and enabled